libraries/AP_MotionController/AP_MotionController.cpp

```diff
@@ -45,13 +45,36 @@ AP_MotionController::~AP_MotionController()
 void AP_MotionController::init(void)
 {
     AP_SerialManager &serial_manager = AP::serialmanager();
-    port = serial_manager.find_serial(AP_SerialManager::SerialProtocol_Volz, 0);
+    port = serial_manager.find_serial(AP_SerialManager::SerialProtocol_ESCTelemetry, 0);
+    roboclaw.init(port, 10000);
     //hal.console->printf("AP_MotionController initialized.\n");
     //gcs().send_text(MAV_SEVERITY_INFO, "AP_MotionController Initialized.");
 }
 
-void AP_MotionController::update(void)
+double AP_MotionController::constrain_map(double x, double in_min, double in_max, double out_min, double out_max)
 {
-    //hal.console->printf("AP_MotionController updated.\n");
-    //gcs().send_text(MAV_SEVERITY_INFO, "AP_MotionController Updated.");
+    //constrain the input value between in_min and in_max
+    if (x < in_min)
```

Comment:
minor formatting thing but normally we have a bracket even for cases where there's just a single line statement afterwards.